### PR TITLE
fix: url change bug

### PR DESCRIPTION
**Pull request by NicolasMassart**
## **Description**

- Remove useless webview props
- clean dead code
- prevent favicon fetch to override cookies policy on the site loaded by webview
- add non regression unit test 

## **Related issues**

Fixes: https://github.com/MetaMask/mobile-planning/issues/1561

## **Manual testing steps**

see initial issue for test instructions

### **Before**

NA

### **After**

NA

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/Views/BrowserTab/index.js
Comment (NicolasMassart) [inline app/components/Views/BrowserTab/index.js:1]: Cleanup of dead code and webview props not existing anymore.
Required to debug and make sure what was failing, but let's keep these changes as it cleans the code a bit.

**Review by witmicko — APPROVED** [review]
verified the fix on the build of this PR, works great.

File: app/components/Views/BrowserTab/index.js [hunk 1]
```diff
@@ -1528,10 +1526,7 @@ export const BrowserTab = (props) => {
                 onMessage={onMessage}
                 onError={onError}
                 onShouldStartLoadWithRequest={onShouldStartLoadWithRequest}
-                sendCookies
-                javascriptEnabled
```
Comment (tommasini) [inline app/components/Views/BrowserTab/index.js:1532]: I see the default value is true: https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#javascriptenabled

**Review by tommasini — APPROVED** [review]
Awesome! LGTM!

**Pull request merged**